cleverhans_tutorials/mnist_blackbox.py

```diff
@@ -91,7 +92,9 @@ def substitute_model(img_rows=28, img_cols=28, nb_classes=10):
     input_shape = (None, img_rows, img_cols, 1)
 
     # Define a fully connected model (it's different than the black-box)
-    layers = [Flatten(),
+    layers = [Conv2D(64, (8, 8), (2, 2), "SAME"),
```

Comment:
The comment above says this model is meant to be fully connected to demonstrate transfer between dissimilar substitute and target, so we shouldn't add a Conv2D layer to this one